src/problem1.rb

```diff
@@ -0,0 +1,9 @@
+class Problem1
+  def sum(array)
+    unless array.nil?
```

Comment:
Very c# like syntax here can you write this with no branching?

_Hint: `[].sum == 0` and `conditional assignment`_